fmt/bytes.ts

```diff
@@ -70,6 +65,74 @@ export interface FormatOptions {
    */
   maximumFractionDigits?: number;
 }
+interface Unit {
+  long: string;
```

Comment:
`long` doesn't seem used?

Reply:
I added long as a clear description and for completeness, similar to [Intl.NumberFormat](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#unit_2).